src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -246,6 +256,26 @@ class Internals(SimpleNamespace):
             short_description="Specify if the editor is console based",
             value=EntryValue(default=True),
         ),
+        Entry(
+            name="exec_command",
+            cli_parameters=CliParameters(short="--excmd"),
```

Comment:
because exec IMO is more of a debugging tool, not a primary subcommand, I didn't want to use `command`, we can only use it once and I'm not sure if we might need it for something more important in the future